controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/SealStreamTask.java

```diff
@@ -72,6 +91,43 @@ public SealStreamTask(final StreamMetadataTasks streamMetadataTasks,
                 });
     }
 
+    private CompletableFuture<Boolean> cancelTransactions(OperationContext context, String scope, String stream) {
+        return streamMetadataStore.getActiveTxns(scope, stream, context, executor)
+                .thenCompose(activeTxns -> {
+                    if (activeTxns == null || activeTxns.isEmpty()) {
```

Comment:
From the current code see that activeTxns cannot be null.

Reply:
yes, but we are being extra cautious with null and empty check so its ok